README.md

```diff
@@ -119,7 +119,7 @@ npx stylelint "**/*.{css,scss}"
 
 You can deploy this project using:
 
--Github
+-Github: https://bohaz.github.io/First-mobile-project/
```

Comment:
- [ OPTIONAL ] Instead of putting your Live Link here, I will suggest that you readd the `Live Demo Section` of the README template so it's easier for users to find the live link to your project :100: 
- [ ] EG.
![image](https://github.com/bohaz/First-mobile-project/assets/102699958/9e5c10e8-59ca-45f9-994f-d24d7324bedd)